viewer/src/app/feature-view/feature-view.component.ts

```diff
@@ -98,7 +113,11 @@ export class FeatureViewComponent implements OnChanges, AfterViewInit {
         return
       }
       case 'BRT': {
-        this.map.addLayer(this.brtLayer())
+        // if (this._projection.visualProjection === 'EPSG:28992') {
```

Comment:
code in comments. 

Wordt `projectionSetRD` nu nog wel gebruikt ergens? Anders zou ik die verwijderen

Reply:
✔️